apollo-rx2-support/src/main/java/com/apollographql/apollo/rx2/Rx2Apollo.java

```diff
@@ -52,7 +52,7 @@ private Rx2Apollo() {
   public static <T> Observable<Response<T>> from(@NotNull final ApolloQueryWatcher<T> watcher) {
     checkNotNull(watcher, "watcher == null");
     return Observable.create(new ObservableOnSubscribe<Response<T>>() {
-      @Override public void subscribe(final ObservableEmitter<Response<T>> emitter) throws Exception {
+      @Override public void subscribe(@NotNull final ObservableEmitter<Response<T>> emitter) {
```

Comment:
+1 for `@NotNull`
+1 for removing the `throws Exception`, I guess it was carried over from the super interface but it's never used in this implementation.